### ISSUE TITLE
Clipping during animations

### ISSUE THREAD

**Issue by mwestphal** [f3d-app/f3d#57]
The bones transformation is not taken into consideration when VTK is computing the clipping planes.
If there is a large scaling transform, clipping artifacts are visible.

**Comment by mwestphal** [comment]
https://gitlab.kitware.com/f3d/f3d/-/issues/250

**Comment by mwestphal** [comment]
I've reworded this issue @Meakk, I thanks it is an important one.

**Comment by mwestphal** [comment]
Some data to test with:
[untitled.zip](https://github.com/f3d-app/f3d/files/10608526/untitled.zip)


stl: 
![stl](https://user-images.githubusercontent.com/3129530/216765950-94c96eb8-913f-4a9a-8bdd-e95bbf1940df.png)

glb:
![glb](https://user-images.githubusercontent.com/3129530/216765963-0af61694-0c03-4e4c-b426-f383b4700377.png)

dae:
![dae](https://user-images.githubusercontent.com/3129530/216765972-82fd09a7-7d9d-46e9-942a-28db79351667.png)

fbx:
![fbx](https://user-images.githubusercontent.com/3129530/216765978-74681071-20a7-4995-a6f2-93cea689a2a0.png)


So there seems to be an issue specific to fbx, and a more generic issue with importers in general

**Comment by mwestphal** [comment]
Do you have an idea for a fix @Meakk ? If not will move it back

**Comment by Meakk** [comment]
I don't have a solution right now, it will need some research

**Comment by mwestphal** [comment]
Unless I'm mistaken, we "just" need to recovert the right bounding box for the data. I think the mapper could compute it and provide it, what do you think @Meakk ?

**Comment by Meakk** [comment]
Yes, we need to override `ComputeBounds()` in the F3D mapper and take into account the bone positions.

**Comment by mwestphal** [comment]
Do you think you could improve the description in order to make it clear how this could be implemented ?
